### ISSUE TITLE
Upgrade @sentry/react-native to 2.2.0

### ISSUE THREAD

**Issue by lc3t35** [expo/sentry-expo#173]
**Summary**

I would like to use "performance monitoring" feature but within expo managed app.
I have the following error at runtime : 

TypeError : undefined is not a constructor (evaluating 'new Sentry.Native.ReactNativeTracing)

**Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!**

managed

**What platform(s) does this occur on?**

iOS

**SDK Version (managed workflow only)**

40

**Environment**

```
Expo CLI 4.1.6 environment info:
    System:
      OS: macOS 10.15.7
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 12.13.1 - /usr/local/bin/node
      Yarn: 1.22.5 - /usr/local/bin/yarn
      npm: 3.10.10 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.9.3 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: iOS 14.4, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
      Android SDK:
        API Levels: 23, 25, 26, 27, 28, 29
        Build Tools: 28.0.3, 29.0.2, 30.0.2
        System Images: android-25 | Google Play Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google Play Intel x86 Atom, android-28 | Google Play Intel x86 Atom_64, android-29 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 3.6 AI-192.7142.36.36.6392135
      Xcode: 12.4/12D4e - /usr/bin/xcodebuild
    npmPackages:
      expo: ^40.0.0 => 40.0.0
      react: 16.13.1 => 16.13.1
      react-native: 0.63.4 => 0.63.4
    npmGlobalPackages:
      expo-cli: 4.1.6
    Expo Workflow: managed
```

**Reproducible demo or steps to reproduce from a blank project**

init Sentry with integrations of ReactNativeTracing

Here is the init code : 

```
import * as Sentry from 'sentry-expo';

Sentry.init({
  dsn:
    'https://.....',
  enableInExpoDevelopment: true,
  debug: true,
  enableAutoSessionTracking: true,
  // Sessions close after app is 10 seconds in the background.
  sessionTrackingIntervalMillis: 10000,
  integrations: [
    new Sentry.Native.ReactNativeTracing({
      tracingOrigins: ['localhost', /^\//, /^https:\/\//],
    }),
  ],
});
```

**Comment by danielamiao** [comment]
Running into the same issue and I think it's because the underlying react native SDK is still on `2.1.0` whereas the react native tracing feature was added in `2.2.0`

![image](https://user-images.githubusercontent.com/2438477/108255764-85511b80-7111-11eb-89eb-4b906420b6d7.png)

@cruzach could we request an upgrade of the underlying SDK from `2.1.0` to `2.2.0`?

**Comment by cruzach** [comment]
sure, will make a note to work on that 👍 

have some other changes in the works currently that are primary focus for `sentry-expo`, but will upgrade sentry SDK afterwards

**Comment by cruzach** [comment]
fixed in https://github.com/expo/sentry-expo/commit/c3ce98db2d967ac761dd9c49f92b6450590b135b, will be releasing a new version as `@next` today with quite a few changes inside of it, so please open up issues for any bugs you find

**Comment by cruzach** [comment]
available in v3.1.0, to get it- run `yarn add sentry-expo@next`
